services/ui-src/src/components/pages/Admin/AdminPage.tsx

```diff
@@ -46,50 +73,17 @@ export const AdminPage = () => {
         <Text>{verbiage.intro.body}</Text>
       </Box>
       <Box sx={sx.currentBannerSectionBox}>
-        <Text sx={sx.sectionHeader}>Current Banner</Text>
+        <Text sx={sx.sectionHeader}>Current Banner(s)</Text>
         {bannerLoading ? (
           <Flex sx={sx.spinnerContainer}>
             <Spinner size="md" />
           </Flex>
         ) : (
           <>
-            <Collapse in={!!bannerData?.key}>
-              {bannerData && (
-                <>
-                  <Flex sx={sx.currentBannerInfo}>
-                    <Text sx={sx.currentBannerStatus}>
-                      Status:{" "}
-                      <span className={bannerActive ? "active" : "inactive"}>
-                        {bannerActive ? "Active" : "Inactive"}
-                      </span>
-                    </Text>
-                    <Text sx={sx.currentBannerDate}>
-                      Start Date:{" "}
-                      <span>{convertDateUtcToEt(bannerData?.startDate)}</span>
-                    </Text>
-                    <Text sx={sx.currentBannerDate}>
-                      End Date:{" "}
-                      <span>{convertDateUtcToEt(bannerData?.endDate)}</span>
-                    </Text>
-                  </Flex>
-                  <Flex sx={sx.currentBannerFlex}>
-                    <Banner status={AlertTypes.INFO} bannerData={bannerData} />
-                    <Button
-                      variant="danger"
-                      sx={sx.deleteBannerButton}
-                      onClick={deleteAdminBanner as MouseEventHandler}
-                    >
-                      {bannerDeleting ? (
-                        <Spinner size="md" />
-                      ) : (
-                        "Delete Current Banner"
-                      )}
-                    </Button>
-                  </Flex>
-                </>
-              )}
+            <Collapse in={allBanners && allBanners?.length > 0}>
+              {allBanners?.map((banner) => BannerPreview(banner))}
             </Collapse>
-            {!bannerData?.key && <Text>There is no current banner</Text>}
+            {!allBanners?.length && <Text>There is no current banner</Text>}
```

Comment:
Suggestion: just a language tweak here, to make sure the user understands we're looking for past & future banners as well.

```suggestion
            {!allBanners?.length && <Text>There are no existing banners</Text>}
```